tests/core/mempool/test_mempool_manager.py

```diff
@@ -159,6 +159,8 @@ def make_test_conds(
             Spend(
                 spend_id,
                 IDENTITY_PUZZLE_HASH,
+                IDENTITY_PUZZLE_HASH,
```

Comment:
does it matter that we use the same for both?

Reply:
no, it's just any value